src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -1268,4 +1278,72 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
     ENDLOOP.
 
   ENDMETHOD.
+
+
+  METHOD confirm_transport_message.
+
+    TYPES: BEGIN OF ty_s_message,
+             id TYPE symsgid,
+             ty TYPE symsgty,
+             no TYPE symsgno,
+             v1 TYPE symsgv,
+             v2 TYPE symsgv,
+             v3 TYPE symsgv,
+             v4 TYPE symsgv,
+           END OF ty_s_message.
+
+    DATA ls_message TYPE ty_s_message.
+
+    FIELD-SYMBOLS: <lt_confirmed_messages> TYPE STANDARD TABLE.
+
+    IF gv_confirm_trans_msg_possible = zif_abapgit_definitions=>c_no.
+      " auto-confirmation of transport messages is not possible in this system.
+      RETURN. ">>>>>>>>>
```

Comment:
What are these for? Why do they increase in numbers for every return statement?

Reply:
you mean the >>> ? I like to put them to RETURN / CONTINUE / CHECK to make the stop of processing at this point more obvious to the eye when looking through code.

Reply:
They already seem quite obvious to me :P
![image](https://github.com/abapGit/abapGit/assets/5270359/37c75794-ec7a-487c-bdf9-48247438b62b)

Reply:
In your IDE yes but not necessarily for others. But i can remove them if it's not a common thing in this project. Should I?
![image](https://github.com/abapGit/abapGit/assets/83953375/d186e1a4-0e3f-422a-a5ad-327e6521f619)

Reply:
we don't use the ">>> in AG, so you can remove them.